atc/api/policychecker/handler.go

```diff
@@ -34,10 +34,15 @@ type policyCheckingHandler struct {
 func (h policyCheckingHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	acc := accessor.GetAccessor(r)
 
+	logger := h.logger.Session("policy-checker")
 	result, err := h.policyChecker.Check(h.action, acc, r)
 	if err != nil {
 		w.WriteHeader(http.StatusBadRequest)
-		fmt.Fprintf(w, "policy check error: %s", err.Error())
+		userMsg := "policy-checker: unreachable or misconfigured"
```

Comment:
I remember that in the fork I mentioned method `SessionName()` of lager, hoping it would return a unique session ID. Since I do not see it here, does it mean that it returns instead "policy-checker" ?

Reply:
Ah... I am almost 100% sure I tried to use it but there were no ID... but honestly now I am confused and don't remember exactly...

From the source code:
https://github.com/cloudfoundry/lager/blob/8d31fc126ae626bbf33d7a05e405a2f4be414345/logger.go#L56
there is no mention of ID.

Reply:
OK, too bad. Thanks.